lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc

```diff
@@ -142,6 +142,30 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
       ),
     );
   }
+  // Add goals/progress.
+  if (module_exists('dosomething_campaign')) {
+    $form['goals'] = array(
+      '#type' => 'fieldset',
+      '#title' => t('Goals Settings'),
+      '#collapsible' => TRUE,
+      '#collapsed' => TRUE,
+    );
+    $form['goals']['goal'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Goal'),
+      '#required' => FALSE,
```

Comment:
I think it's not required by default. Could you please check it and remove if yes?